build/Dockerfile.gcloud-kubernetes

```diff
@@ -0,0 +1,7 @@
+FROM gcr.io/cloud-builders/gcloud
+
+RUN gcloud config set compute/zone us-east1-b
+RUN gcloud components install kubectl -q
```

Comment:
Maybe just add this to `Dockerfile.mvn-gcloud` so that we wouldn't have to build yet another image for testing?